pkgs/development/python-modules/pdoc-pyo3-sample-library/default.nix

```diff
@@ -39,5 +40,6 @@ buildPythonPackage rec {
     homepage = "https://github.com/mitmproxy/pdoc-pyo3-sample-library";
     license = lib.licenses.mit;
     maintainers = [ lib.maintainers.pbsds ];
+    broken = stdenv.isDarwin; # -liconv
```

Comment:
Instead add `libiconv` to `buildInputs` on Darwin.

Reply:
Huh, TIL about `darwin.libiconv`